fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java

```diff
@@ -164,6 +164,10 @@ public void handleFinishAlterTask() throws MetaNotFoundException {
 
         db.writeLock();
```

Comment:
How about validate if database is dropped when calling db.xxLock()?

Reply:
done